### PR TITLE
Script for Tracking Progress

### PR DESCRIPTION
**Pull request by Ebuka-Chibueze**
The script for [issue-521](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/521).

RawGit: [link](http://rawgit.com/open-learning-exchange/open-learning-exchange.github.io/Ebuka-Chibueze-issue-521/pages/track-progress.html)

### CODE REVIEW

**Comment by kylemathias** [comment]
@Ebuka-Chibueze Nice work. I am getting some errors with API search. 
image

**Comment by Ebuka-Chibueze** [comment]
@kylemathias If you're concerned about the appearance, I don't think that's an error.

**Comment by dogi** [comment]
@Ebuka-Chibueze very awesome

now we need to check it ;)

**Comment by Ebuka-Chibueze** [comment]
@mappuji  @kylemathias  
I just realized how API button isn't very user friendly. I'll definitely change the directions.
Also I'll see if I can find a neater way to display it.

File: pages/track-progress.html [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+		console.log(object)
+	
+
+	
+// Initialize a request
+request.open('get', object)
+request.send()
+		request.onreadystatechange = function() {
+    if (this.readyState == 4 && this.status == 200) {
+      alert("Number of PRs:"+request.responseText.substr(18,3))
```
Comment (Ebuka-Chibueze) [inline pages/track-progress.html]: @paulbert Thank you , now we don't have to worry about commas popping up in the alert.

**Comment by mappuji** [comment]
Yes, overall the UI is very good @Ebuka-Chibueze .
Btw, how is the progress? Is there is any problem when following up @paulbert and my suggestion?

**Comment by Ebuka-Chibueze** [comment]
@mappuji  I decided to keep it simple and only use one button. And I was able to follow Paul's advice.

**Comment by mappuji** [comment]
Hi all
I wonder if there is already link pointing to this [work](http://open-learning-exchange.github.io/pages/track-progress.html)?

summon @dogi and @Ebuka-Chibueze

**Comment by sarah-lu102** [comment]
@mappuji I found a link in the first steps page. 
image

**Comment by mappuji** [comment]
Cool @sarah-lu102

**Pull request merged**